packages/http-client-csharp/generator/Microsoft.Generator.CSharp.Input/src/InputTypes/InputIntrinsicType.cs

```diff
@@ -1,8 +1,10 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 namespace Microsoft.Generator.CSharp.Input
 {
+    // TODO -- after we have adopted getAllModels in our emitter, we no longer need to have this type.
```

Comment:
Yeah we should align our input types with types from TCGC as much as possible otherwise we will always maintain a complicated converter in our emitter.
I have this TODO because I know TCGC is handling the intrinsic types differently from what we are doing right now.